pyvrp/Model.py

```diff
@@ -128,7 +130,12 @@ def add_client(
         return client
 
     def add_depot(
-        self, x: int, y: int, tw_early: int = 0, tw_late: int = 0
+        self,
+        x: int,
+        y: int,
+        tw_early: int = 0,
+        tw_late: int = 0,
+        release_time: int = 0,
```

Comment:
`tw_early` and `release_time` essentially do the same thing. Not sure if we should include release time here.

Reply:
I don't know what a release time for the depot would mean. Let's just use tw early here and assume the depot release time is zero.